manual/python/index.md

```diff
@@ -1,70 +1,95 @@
 ---
-title: Python interface PyROOT
+title: "Python interface: PyROOT"
 layout: single
 sidebar:
   nav: "manual"
 toc: true
 toc_sticky: true
 ---
 
-ROOT provides an interface to Python, which is enabled via a set of bindings called PyROOT.<br>
-PyROOT is enabled by default in ROOT.
+ROOT can also be used from Python thanks to its Python-C++ bindings, called **PyROOT**.  PyROOT allows to access all the ROOT functionality from Python while, at the same time, benefiting from the performance of the ROOT C++ libraries.
```

Comment:
Probably we should point out more prominently that pyroot is not ROOT on Python but **generic** and totally automatic python bindings for C++ libraries. There's a huge difference! I think it's really hard to understand what PyROOT is capable to do, or let's say cppyy?

Reply:
Sure! I will make that clear in this intro package, thanks. I also have a section (not written yet) that will demonstrate how to dynamically load C++ libraries and call into them right away from Python.